lms/djangoapps/instructor/enrollment.py

```diff
@@ -267,6 +269,15 @@ def reset_student_attempts(course_id, student, module_state_key, requesting_user
 
     if delete_module:
         module_to_reset.delete()
+        if block and block.has_score:
+            SCORE_CHANGED.send(
+                sender=None,
+                points_possible=block.max_score,
```

Comment:
I just pushed some changes that bind the user to the block. I still need to address this though. For posterity, per earlier discussions offline, we do need to give a value for `points_possible' because of this handler for `SCORE_CHANGED`: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/lti_provider/tasks.py#L40